### DISCUSSION TITLE
v6.0.0

### DISCUSSION THREAD

**Discussion by tadashi-aikawa** [tadashi-aikawa/obsidian-another-quick-switcher#78] [Announcements]
# Released [v6.0.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/6.0.0) 🚀

**Comment by tadashi-aikawa** [comment]
Released [v6.0.0-beta1](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/6.0.0-beta1) 🚀

**Comment by tadashi-aikawa** [comment]
v6.0.0-beta4 will release it. Progress rate: 65% ✨ 

![image](https://user-images.githubusercontent.com/9500018/183290747-e387b404-a947-4ae5-b582-16da3a3aea5b.png)

**Reply by FelipeRearden** [reply]
> v6.0.0-beta4 will release it. Progress rate: 65% ✨ 

💙 👀

**Reply by tadashi-aikawa** [reply]
Improve layout and settings ✨ 

![image](https://user-images.githubusercontent.com/9500018/183299451-b5ca05b3-a6aa-4917-986a-ac12a6e9a5e9.png)

**Comment by tadashi-aikawa** [comment]
Released [v6.0.0-beta8](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/6.0.0-beta8) 🚀

There is a breaking change related to a "default input" behavior.

https://user-images.githubusercontent.com/9500018/184126682-0a432bf3-5083-41a1-b3df-7948c5b79328.mp4

**Comment by tadashi-aikawa** [comment]
v6.0.0-beta9 is coming soon... `Search by` option in custom search commands.

![image](https://user-images.githubusercontent.com/9500018/184181427-caab673a-5fdf-4145-9d43-a993c8cbf6ce.png)
![image](https://user-images.githubusercontent.com/9500018/184181437-7539dabd-9396-4f1f-9836-659b98a489b0.png)

**Reply by tadashi-aikawa** [reply]
Released 🚀

**Comment by tadashi-aikawa** [comment]
v6.0.0-beta10 is coming soon... 

https://user-images.githubusercontent.com/9500018/184470557-cd331c28-0265-4490-9125-a8bc660e8346.mp4

![image](https://user-images.githubusercontent.com/9500018/184470524-15b7d804-afbe-48a0-800f-70467f7bed31.png)

**Reply by tadashi-aikawa** [reply]
Improve settings UI ✨ 

https://user-images.githubusercontent.com/9500018/184473422-a85df7a1-ced9-4a06-8ca4-1915b4b9a5e9.mp4

**Reply by FelipeRearden** [reply]
The new UI on MODAL and SETTINGS in `beta-10` are absolutely spectacular. This is amazing @tadashi-aikawa !!!!

You are giving us a world class In-file Content/File Search in Obsidian that is exactly what we missed. You are a legend @tadashi-aikawa  🥇🥇🥇

**Comment by tadashi-aikawa** [comment]
Released v6.0.0-beta11 🚀 

- #38
- #80
- Fix layout for mobile and others

**Comment by tadashi-aikawa** [comment]
Released v6.0.0-beta13 🚀

> **Note**
> **This version is a release candidate. So I'll release v6.0.0 tomorrow if there are no critical bugs 🎐**

This version improves Grep feature.

- Change the trigger of grep to manually
- Keep the last results and selected to restore when it opens the next time
- Improve performance

https://user-images.githubusercontent.com/9500018/184532438-12865122-6ead-4501-9d96-786aa68e60ff.mp4
